frontend/src/containers/Ledger.js

```diff
@@ -221,16 +227,15 @@ export function rejectExp(expenseId) {
  */
 export function payExp(expenseId) {
   const {
-    collective,
     fetchPendingExpenses,
     fetchTransactions,
     payExpense,
     notify
   } = this.props;
 
-  return payExpense(collective.id, expenseId)
-    .then(() => fetchPendingExpenses(collective.slug))
-    .then(() => fetchTransactions(collective.slug))
+  return payExpense(this.collective.id, expenseId)
+    .then(() => fetchPendingExpenses(this.collective.slug))
```

Comment:
Done but without the details. That's more complicated because the api call doesn't return the details of the approved expense.